src/components/Display.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <p>{result}</p>
+    );
+  }
+}
+
+Display.defaultProps = {
+  result: 0,
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
```

Comment:
- [ ] Since the proptype is defined as a String, it needs to make the default value as a string, not a number
```JS
Display.defaultProps = {
  result: '0',
};

```

Reply:
As you see in the code, `'0'` is a string not a number.